lib/galaxy/tools/actions/__init__.py

```diff
@@ -738,8 +737,6 @@ def handle_output(name, output, hidden=None):
         # execute immediate post job actions and associate post job actions that are to be executed after the job is complete
```

Comment:
I'd also remove `job_params` from the list of paramaters of the `execute()` method above.